java/integTest/src/test/java/glide/SharedClientTests.java

```diff
@@ -48,13 +48,15 @@ public static void init() {
                 GlideClusterClient.createClient(commonClusterClientConfig().requestTimeout(10000).build())
                         .get();
         clients = List.of(Arguments.of(standaloneClient), Arguments.of(clusterClient));
-        assertTrue(!clusterClient.getStatistics().isEmpty());
-        assertEquals(
-                clusterClient.getStatistics().size(), 2); // we expect 2 items in the statistics map
+    }
 
-        assertTrue(!standaloneClient.getStatistics().isEmpty());
-        assertEquals(
-                standaloneClient.getStatistics().size(), 2); // we expect 2 items in the statistics map
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void validate_statistics(BaseClient client) {
```

Comment:
im not sure why this changed...

Reply:
@eifrah-aws added these checks to init. If they fail - no tests executed.